python/cudf/cudf/tests/test_avro_reader_fastavro_integration.py

```diff
@@ -22,9 +22,17 @@
 import pytest
 
 import cudf
+from cudf.core._compat import PANDAS_GE_220
 from cudf.testing._utils import assert_eq
 from cudf.testing.dataset_generator import rand_dataframe
 
+if PANDAS_GE_220:
+    pytestmark = pytest.mark.filterwarnings(
+        "ignore",
+        category=DeprecationWarning,
+        message="Passing a BlockManager to DataFrame is deprecated",
```

Comment:
cudf never deals with the pandas block manager right? If not, I would just globally ignore this in `python/cudf/cudf/tests/pytest.ini` with `ignore:Passing a BlockManager to DataFrame is deprecated:DeprecationWarning`

Reply:
We don't deal with it, it shows up every time when we do:
```
arrow_table = pa.table(...)
arrow_table.to_pandas()
```

Reply:
Done, moved the filter to `pytest.ini`